### PR TITLE
Add a try except

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

**Comment by isuruf** [comment]
why?

**Comment by beckermr** [comment]
Other options will be added later and we want to catch exceptions for all of them.

**Comment by isuruf** [comment]
Wouldn't they need a separate message?

**Comment by beckermr** [comment]
IMO for this error, no.

**Comment by isuruf** [comment]
Doesn't make sense to me, but I've made the change

File: conda_forge_tick/migrators/version.py [hunk 1]
```diff
@@ -636,55 +636,60 @@ def pr_body(self, feedstock_ctx: FeedstockContext) -> str:
...
                     )
-            else:
-                hint += (
-                    "Analysis of the source code shows **no** discrepancy between"
-                    " the library's imports and the package's stated requirements in the meta.yaml."
-                )
+                body += hint
+        except Exception:
+            hint = f"\n\nDependency Analysis\n--------------------\n\n"
+            hint += "We couldn't run dependency analysis due to an error.\n"
```
Comment (beckermr) [inline conda_forge_tick/migrators/version.py:692]:
```suggestion
            hint += (
                "We couldn't run dependency analysis due to an error. Make sure the "
                "recipe and the source code download URL are correct.\n"
            )
```

Comment (isuruf) [reply conda_forge_tick/migrators/version.py]: That's a bad description for the original issue that prompted this.

Comment (beckermr) [reply conda_forge_tick/migrators/version.py]: ack I missed the patch :/

**Comment by isuruf** [comment]
Can you please preserve commits when you merge?

**Comment by beckermr** [comment]
Ohhh sorry! I forgot you like don't like squash and merge. I will make sure to do that next time!

**Comment by CJ-Wright** [comment]
Thank you very much for the fix!

**Pull request merged**